### ISSUE TITLE
PHP Interop Docker Image Fails to Build

### ISSUE THREAD

**Issue by gnossen** [grpc/grpc#19437]
```
+ php -d extension=ext/grpc/modules/grpc.so /usr/local/bin/composer install
Do not run Composer as root/super user! See https://getcomposer.org/root for details
Loading composer repositories with package information
Updating dependencies (including require-dev)
  [Composer\Downloader\TransportException]
  The "https://packagist.org/p/provider-2015%2418c42deccda75e24d7425d30ec42196c69f9ad7e0550902d126809c4f4fd276c.json" file could not be downloaded (HTTP/1.1 404 Not Found)
install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--] [<packages>]...
2019-06-21 14:15:01,894 FAILED: build_docker_php [ret=1, pid=11976, time=650.8sec]
```

[Test run.](https://source.cloud.google.com/results/invocations/ff7b01b3-0259-452c-879a-dcb5fe23adb1/log)

**Comment by stanley-cheung** [comment]
Looks like it's a flake? From the log looks like composer is temporarily messed up and that's out of our control.

**Comment by hcaseyal** [comment]
Happened again
https://source.cloud.google.com/results/invocations/f2127060-5f99-45b1-9e51-b22591564367/targets/grpc%2Fcore%2Fpull_request%2Flinux%2Fgrpc_interop_tocloud/log

**Comment by stanley-cheung** [comment]
An attempt to fix in #20819
